new-client/src/components/App.js

```diff
@@ -485,7 +485,9 @@ class App extends React.PureComponent {
 
   renderSearchComponent() {
     // FIXME: We should get config from somewhere else now when Search is part of Core
-    if (this.appModel.plugins.search) {
+    const renderSearchComponent = !this.appModel.plugins.search.options
+      .renderElsewhere;
+    if (renderSearchComponent && this.appModel.plugins.search) {
```

Comment:
Did you test this with a config without a search-component? I think this.appModel.plugins.search.options will be undefined in that case, causing a crash?

Reply:
Well spotted. I will update.